src/basics.md

```diff
@@ -805,6 +806,45 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-hmac]: #ex-hmac
+<a name="ex-hmac"></a>
+## Sign and verify a message with HMAC digest
+
+[![ring-badge]][ring] [![data-encoding-badge]][data-encoding] [![cat-cryptography-badge]][cat-cryptography]
+
+Creates a [`hmac::Signature`] of a string then verifies the signiture is correct.
```

Comment:
I would suggest adding other hyperlinked identifiers to the description for the benefit of the reader.